test/README.md

```diff
@@ -108,12 +108,20 @@ tests will fail. If this happens, remove the cache directory (and make
 sure bitcoind processes are stopped as above):
 
 ```bash
-rm -rf cache
+rm -rf test/cache
 killall bitcoind
 ```
 
 ##### Test logging
 
+The test framework logs to two files: 
+ * `<test data directory>/test_framework.log` 
+ * `<test data directory>/node<number>/regtest/debug.log`. 
+ 
+The `<test data directory>` path is by default set to the temporary directory of
+your OS, and its full path is written as the very first line of the test output.
```

Comment:
Suggestion: "its full path is provided in the first line of the test output."

Reply:
> Suggestion: "its full path is provided in the first line of the test output."

Agreed, better! Fixed.